actions/approve-pr/README.md

```diff
@@ -0,0 +1,45 @@
+# Approve pull request action
+
+This action approves a pull request and optionally enables automerge. See the [create pull request action](../create-pr/README.md) for a ready to use action that can create a pull request.
+
+## Action inputs
+
+| Input                 | Description                                | Default               |
+| --------------------- | ------------------------------------------ | --------------------- |
+| `pull_request_number` | The number of the pull request to approve. | _required_            |
+| `auto_merge`          | Enable auto-merge for the pull request.    | `false`               |
+| `token`               | Access token to manage the pull request.   | `${{ GITHUB_TOKEN }}` |
```

Comment:
might be worth to point out that github does not trigger CI events if those events are triggered by the base `GITHUB_TOKEN` and on which case the user would need to add their own token if they would like to have the CI events triggered

Reply:
Done, thanks